### DISCUSSION TITLE
Should I learn App Dev.

### DISCUSSION THREAD

**Discussion by adhyan-jain** [community/community#131927] [Other Feature Feedback, Questions, & Ideas] [answered]
I am starting college soon and will be pursuing AI & ML someone suggested that I learn app development cause it would help me during projects as I can just make an app on whatever project I am working on.
Since I am just starting, I was thinking of exploring different fields but would it be better if I just focus on AI & ML as I have already taken that in college or should I learn App Dev.
Basically I am asking if it is useful or not.
Finally if I should learn app dev. which would be better Java or Kotlin or any other PL.
By the way don’t know if its helps but I know a little bit of python.
Thanks in advance

**Comment by ghost** [comment]
Application development is completely optional, but it will give you hands-on experience with programming concepts and techniques.
If you decide to get into app development, it might be better to learn Kotlin.

**Reply by adhyan-jain** [reply]
Thanks for the advice.
Is there any reason you are recommending Kotlin?
And what do you think would be more beneficial, learning further about the thing you already know or learning new things to amplify your results?

**Reply by ghost** [reply, marked as answer]
I recommend Kotlin because it is a versatile and modern programming language that offers many benefits and strong support for functional programming, in addition, it has a fairly simple syntax and it will be easier for you to understand how everything works. As for the second question, it ultimately depends on your goals and interests. If you want to become an expert in a certain field, expanding your knowledge in that area can be more rewarding. However, learning new things can also be good for expanding your skills.

**Reply by adhyan-jain** [reply]
Thanks for the advice
